src/model/kube/statusFetcher_test.go

```diff
@@ -0,0 +1,30 @@
+package kube
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestCheckHealth_AllHealty(t *testing.T) {
```

Comment:
Typo in test method:

```suggestion
func TestCheckHealth_AllHealthy(t *testing.T) {
```